src/controller/authController.ts

```diff
@@ -0,0 +1,24 @@
+import {Request, Response, Application, response} from "express";
+import { Login } from "../model/auth";
+const authService = require("../service/authService");
```

Comment:
Should be an import and not a require. 

Also check to see if the imports should be import or import type